### PR TITLE
Pnda 2708: add pip index URL in order to enable offline installation

### PR DESCRIPTION
**Pull request by jgarnier**

### CODE REVIEW

File: scripts/base_install.sh [hunk 1]
```diff
@@ -76,7 +76,23 @@ roles: [${ROLES}]
 EOF
 fi
 
+if [ "$pnda_mirror$" != "$" ] ; then
```
Comment (jeclarke) [inline scripts/base_install.sh]: pnda_mirror is mandatory so take out the conditional

File: scripts/base_install.sh [hunk 2]
```diff
@@ -76,7 +76,23 @@ roles: [${ROLES}]
...
+index-url=$PIP_INDEX_URL
+trusted-host=$TRUSTED_HOST
+EOF
+cat << EOF > /root/.pydistutils.cfg
+[easy_install]
+index_url=$PIP_INDEX_URL
+EOF
+fi
+
+if [ "x$DISTRO" == "xubuntu" ]; then
```
Comment (jeclarke) [inline scripts/base_install.sh]: this command can run on all rhel or ubuntu, so take out conditional?

File: scripts/base_install.sh (hunk 1 above)
Comment (jgarnier) [reply scripts/base_install.sh]: agree I've put it for a simple check but can now be removed

File: scripts/base_install.sh (hunk 2 above)
Comment (jgarnier) [reply scripts/base_install.sh]: yes sure

**Comment by jeclarke** [comment]
thanks @jgarnier !

**Pull request merged**